kategory/src/main/kotlin/kategory/instances/ConstInstances.kt

```diff
@@ -16,6 +17,9 @@ interface ConstInstances<A> :
 
     override fun <T, U> foldR(fa: HK<ConstKindPartial<A>, T>, lb: Eval<U>, f: (T, Eval<U>) -> Eval<U>): Eval<U> = lb
 
+    override fun <G, T, U> traverseFilter(fa: HK<ConstKindPartial<A>, T>, f: (T) -> HK<G, Option<U>>, GA: Applicative<G>):
+            HK<G, HK<ConstKindPartial<A>, U>> = fa.ev().traverseFilter(f, GA)
```

Comment:
while method args is ok to take the most generic representation of the HK or Kind aliases in return type we should return the most concrete compatible In this case `HK<ConstKindPartial<A>, U>` can just be `Const<A, U>` same for all other instance methods, return always the most concrete type so users don't have to use `ev()` to extract values from the HK representation.

Reply:
I'm going to change also in `traverse` method 👍